### PR TITLE
[MIG] stock_picking_delivery_info_computation: Migration to v12

### PR DESCRIPTION
**Pull request by victor-ditay**

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Code review

File: stock_picking_delivery_info_computation/README.rst [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to *Inventory > All Transfers* and pick one.
```
Comment (rafaelbn) [inline stock_picking_delivery_info_computation/README.rst]: All transfers* is not valid.  Only draft, waiting and ready. If the transfer is Done or cancelled the button doesn't appears

**Review by rafaelbn — APPROVED** [review]
Functional test :+1: 

Just review one point in README thanks

File: stock_picking_delivery_info_computation/README.rst (hunk 1 above)
Comment (pedrobaeza) [reply stock_picking_delivery_info_computation/README.rst]: This is because README must be adapted to v12

**Comment by victor-ditay** [comment]
Changes done @rafaelbn

**Comment by victor-ditay** [comment]
@pedrobaeza Commit cherry picked from #213

File: stock_picking_delivery_info_computation/views/delivery_view.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
         <field name="model">stock.picking</field>
         <field name="inherit_id" ref="delivery.view_picking_withcarrier_out_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='number_of_packages']" position="after">
```
Comment (pedrobaeza) [inline stock_picking_delivery_info_computation/views/delivery_view.xml]: We should add this field as it has been removed and it's important for the aim on the module: delivery info.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
One important thing for keeping information consistent

**Comment by victor-ditay** [comment]
Changes done @pedrobaeza

**Review by pedrobaeza — APPROVED** [review]

**Pull request closed without merging**